### DISCUSSION TITLE
c compiler not found

### DISCUSSION THREAD

**Discussion by ata2151** [mortbopet/Ripes#87] [Q&A]
I keep getting this message when trying to use c code: 
`No C compiler set.`
`Provide a path to a valid C compiler under:`
 `Edit->Settings->Editor`

Note that I have already added the compiler path in the Editor.

**Comment by mortbopet** [comment]
If using the prebuilt version 2.1, make sure that the compiler path does not contain any spaces (should be fixed if you are building from master).
Also, make sure that you've gone through https://github.com/mortbopet/Ripes/wiki/Building-and-Executing-C-programs-with-Ripes.
